vnext/Microsoft.ReactNative/IReactContext.idl

```diff
@@ -3,11 +3,13 @@
 
 import "IJSValueWriter.idl";
 
+#include "../Include/NamespaceRedirect.h"
+
 namespace Microsoft.ReactNative {
 
   [webhosthidden]
   interface IReactContext {
-    void DispatchEvent(Windows.UI.Xaml.FrameworkElement view, String eventName, JSValueArgWriter eventDataArgWriter);
+    void DispatchEvent(WUXMUX_TYPE(FrameworkElement) view, String eventName, JSValueArgWriter eventDataArgWriter);
```

Comment:
>WUXMUX_TYPE [](start = 23, length = 11)

It looks too cryptic.

Reply:
I had gotten some error when doing the concatenated XAML_NAMESPACE.FrameworkElement but to be fair this was at like 2am... re-tried now and it works fine so I'm reverting back to using XAML_NAMESPACE.FrameworkElement instead of building the concatenation